requirements-release.txt

```diff
@@ -7,3 +7,4 @@ wheel
 setuptools
 twine
 parameterized
+google-re2
```

Comment:
This should be moved to requirements.txt with `google-re2[text]`. See https://pip.pypa.io/en/stable/reference/requirement-specifiers/#overview.